src/zc/buildout/buildout.py

```diff
@@ -2115,7 +2140,7 @@ def main(args=None):
         command = args.pop(0)
         if command not in (
             'install', 'bootstrap', 'runsetup', 'setup', 'init',
-            'annotate',
+            'annotate', 'query',
```

Comment:
I wish there was a way not to have to repeat command names here.

Perhaps a simple `@command` decorator that appends method names to a list of commands...

Anyway, just a thought, not an actual request for change.

Reply:
Nice idea !